source/unified-test-format/unified-test-format.rst

```diff
@@ -1251,6 +1280,67 @@ expectedCmapEvent
   MUST be an empty document as all fields in this event are non-deterministic.
 
 
+expectedSdamEvent
+`````````````````
+
+The structure of this object is as follows:
+
+.. _expectedEvent_serverDescriptionChangedEvent:
+
+- ``serverDescriptionChangedEvent``: Optional object. Assertions for one or more
```

Comment:
rather than introduce a special test-only `ServerMarkedUnkownEvent`, I opted to just use a vanilla `ServerDescriptionChangedEvent` and encode what a marked unknown event is in the test assertions. This seemed more generalizable.